arcgis-ios-sdk-samples/Layers/Query with CQL filters/QueryWithCQLFiltersSettingsViewController.swift

```diff
@@ -0,0 +1,168 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+protocol QueryWithCQLFiltersSettingsViewControllerDelegate: AnyObject {
+    func settingsViewController(_ controller: QueryWithCQLFiltersSettingsViewController, didCreate queryParameters: AGSQueryParameters)
+}
+
+class QueryWithCQLFiltersSettingsViewController: UITableViewController {
+    // MARK: Storyboard views
+    
+    /// The cell for where clause options.
+    @IBOutlet var whereClauseCell: UITableViewCell! {
+        didSet {
+            whereClauseCell.detailTextLabel?.text = selectedWhereClause
+        }
+    }
+    /// The text field for max features input.
+    @IBOutlet var maxFeaturesTextField: UITextField! {
+        didSet {
+            // Add a toolbar to dismiss keyboard.
+            let toolbar = UIToolbar()
+            let items: [UIBarButtonItem] = [
+                .init(barButtonSystemItem: .flexibleSpace, target: nil, action: nil),
+                .init(barButtonSystemItem: .done, target: self, action: #selector(inputAccessoryViewDoneButtonTapped(_:)))
+            ]
+            toolbar.setItems(items, animated: true)
```

Comment:
No need for this to be animated as it isn't even on screen yet.

```suggestion
            toolbar.setItems(items, animated: false)
```